src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/Offender.java

```diff
@@ -242,4 +243,12 @@
                 .filter(PrisonOffenderManager::isActive)
                 .findFirst();
     }
+    public static class OffenderBuilder {
```

Comment:
Only used in tests that use builders(). Ensure POM list is mutable so we can amend that list